src/main/java/com/github/manolo8/darkbot/utils/login/LoginUtils.java

```diff
@@ -81,21 +84,32 @@ public static LoginData performAutoLogin(StartupParams params) {
         loginData.setCredentials(username, password);
 
```

Comment:
```suggestion
        if (isSidLogin) loginData.setSid(sid, server + ".darkorbit.com");
```
i'd just set it from the start, since this cannot produce an exception